src/components/portal/portal.tsx

```diff
@@ -143,33 +146,30 @@ export class Portal {
     private showContainer() {
         this.container.style.opacity = '1';
     }
+
     private styleContainer() {
         const rect: any = this.host.getBoundingClientRect();
-        const viewportHeight = this.getViewportHeight();
-        const containerHeight = viewportHeight - rect.y;
 
-        this.container.style.height = `${containerHeight}px`;
-        this.container.style.display = 'block';
-        if (!this.visible) {
+        if (this.visible) {
+            this.container.style.display = 'block';
+            this.container.style.opacity = '1';
```

Comment:
Do we need this opacity stuff still?

Reply:
I think this is a good fix but it's no longer related to this PR. I say we keep it.

Reply:
Ah, yeah, this was a fix for if the portal is created with `visible` already `true` right? Yeah, then keep it 👍